01-JS-DOM-APIs/Practice1-1/Script/script.js

```diff
@@ -0,0 +1,43 @@
+function load () {
+  console.log('load'); //Only to know the page is loaded
+  let change = document.getElementById("changeclass");
+  change.style.transition = "all 3s ease-in-out";
+  change.style.opacity = 1;
+}
+
+
+function configAjax (methodHttp, url, asyncronic){
```

Comment:
when you do *this.methodHttp = methodHttp* you are assigning a methodHttp variable to the global scope, you should try to avoid working with the global scope when possible to make sure that you don't cause unexpected behavior, instead do something like this:
````
var ajaxConfig = {};

ajaxConfig.methodhttp = methodHttp;
ajaxConfig.url = url;
ajaxConfig.asyncronic = asyncronic;

return ajaxConfig;
```` 

or instead of a configAjax function you can just define an object with the attributes you need.